### PR TITLE
fix: re-implement List early render

### LINKED ISSUE #23925: [Bug]: Dialog doesn't focus on list
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.22622
    CPU: (12) x64 Intel(R) Xeon(R) CPU E5-1650 v4 @ 3.60GHz
    Memory: 9.79 GB / 31.92 GB
  Browsers:
    Chrome: 103.0.5060.114
    Edge: Spartan (44.22621.290.0), Chromium (103.0.1264.37), ChromiumDev (104.0.1293.5)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codepen.io/mbest098/pen/WNzozwO

## Actual Behavior
Initial focus in dialog goes after the list

## Expected Behavior
A focusable list should receive focus if it's the first item in the dialog

### Logs

_No response_

### Requested priority

High

### Products/sites affected

https://account.microsoft.com/family/

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### LINKED ISSUE #26899: [Bug]: List rendering change causes rendering bug in TilesList
### Library

React / v8 (@fluentui/react)

### System Info

```shell
n/a
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://fluentuipr.z22.web.core.windows.net/heads/master/react-experiments/demo/index.html#/examples/tileslist

## Actual Behavior

`TilesList` rendering is broken.

### Repro steps

1. Navigate to: https://fluentuipr.z22.web.core.windows.net/heads/master/react-experiments/demo/index.html#/examples/tileslist
2. View the first example

tileslist_broken_render

## Expected Behavior

`TilesList` rendering is not broken.

tileslist_correct_render

### Logs

_No response_

### Requested priority

Blocking

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by spmonahan**
## Previous Behavior

Lists cannot be focused in certain situations like when displayed in a Dialog.  See #23925 for details.

## New Behavior

When using the `renderEarly` flag List can be focused when displayed in a Dialog. The root of the issue is that, by default, List takes multiple passes to render its content and the actual list contents are not available after the first React `render()` call. Because nothing is rendered on the first pass `FocusZone` does not "see" the List and will not focus it. This change allows users to opt into rendering early to address this issue.

Previously this early render behavior was made the default for List (see: #25331) but that resulted in a regression for TilesList (see: #26899). Because of the complexity of the controls in question I've decided to make the new behavior opt-in for users that need it.

## Related Issue(s)

- Fixes #23925
- Fixes #26899

## Depends on

- #26969 (don't merge the List change until we have TilesList VR tests running)

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/18c342bc12f318d4c245d9a35a47f1d93a39d1ec'>18c342bc12f318d4c245d9a35a47f1d93a39d1ec</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

File: change/@fluentui-react-d174ae16-7e82-4494-b026-d256fe1eb378.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-d174ae16-7e82-4494-b026-d256fe1eb378.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27002&commitId=f3469b01cd5c8d3a6956f85236384aadd6060b9b&env=prod)

✅ There was 0 screenshots added, 4 screenshots removed, 1041 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/BaseButton-mount_base.html">1196</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/BaseButton-mount.html">1191</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Breadcrumb-mount_base.html">2804</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Breadcrumb-mount.html">2821</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Checkbox-mount_base.html">2612</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Checkbox-mount.html">2620</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/CheckboxBase-mount_base.html">2387</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/CheckboxBase-mount.html">2375</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/ChoiceGroup-mount_base.html">4287</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/ChoiceGroup-mount.html">4317</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/ComboBox-mount_base.html">1167</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/ComboBox-mount.html">1196</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/CommandBar-mount_base.html">9271</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/CommandBar-mount.html">9295</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/ContextualMenu-mount_base.html">12325</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/ContextualMenu-mount.html">12415</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/DefaultButton-mount_base.html">1349</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/DefaultButton-mount.html">1378</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/DetailsRow-mount_base.html">3238</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/DetailsRow-mount.html">3259</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/DetailsRowFast-mount_base.html">3253</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/DetailsRowFast-mount.html">3210</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/DetailsRowNoStyles-mount_base.html">3138</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/DetailsRowNoStyles-mount.html">3104</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Dialog-mount_base.html">2937</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Dialog-mount.html">3001</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/DocumentCardTitle-mount_base.html">577</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/DocumentCardTitle-mount.html">599</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Dropdown-mount_base.html">3160</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Dropdown-mount.html">3166</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/FocusTrapZone-mount_base.html">1986</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/FocusTrapZone-mount.html">1962</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/FocusZone-mount_base.html">1949</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/FocusZone-mount.html">1910</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/GroupedList-mount_base.html">47268</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/GroupedList-mount.html">52946</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/GroupedList-virtual-rerender_base.html">22201</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/GroupedList-virtual-rerender.html">22540</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">80822</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/GroupedList-virtual-rerender-with-unmount.html">80907</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/GroupedListV2-mount_base.html">576</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/GroupedListV2-mount.html">558</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/GroupedListV2-virtual-rerender_base.html">537</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/GroupedListV2-virtual-rerender.html">541</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">568</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">564</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/IconButton-mount_base.html">1827</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/IconButton-mount.html">1801</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Label-mount_base.html">747</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Label-mount.html">766</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Layer-mount_base.html">4208</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Layer-mount.html">4206</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Link-mount_base.html">855</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Link-mount.html">873</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/MenuButton-mount_base.html">1600</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/MenuButton-mount.html">1638</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/MessageBar-mount_base.html">2367</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/MessageBar-mount.html">2384</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Nav-mount_base.html">3048</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Nav-mount.html">3062</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/OverflowSet-mount_base.html">1393</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/OverflowSet-mount.html">1390</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Panel-mount_base.html">2499</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Panel-mount.html">2526</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Persona-mount_base.html">1276</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Persona-mount.html">1298</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Pivot-mount_base.html">1519</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Pivot-mount.html">1515</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/PrimaryButton-mount_base.html">1491</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/PrimaryButton-mount.html">1499</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Rating-mount_base.html">6918</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Rating-mount.html">6934</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/SearchBox-mount_base.html">1509</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/SearchBox-mount.html">1511</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Shimmer-mount_base.html">2917</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Shimmer-mount.html">2917</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Slider-mount_base.html">2135</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Slider-mount.html">2113</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/SpinButton-mount_base.html">4217</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/SpinButton-mount.html">4255</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Spinner-mount_base.html">836</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Spinner-mount.html">823</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/SplitButton-mount_base.html">2870</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/SplitButton-mount.html">2888</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Stack-mount_base.html">850</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Stack-mount.html">873</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/StackWithIntrinsicChildren-mount_base.html">2245</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/StackWithIntrinsicChildren-mount.html">2275</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/StackWithTextChildren-mount_base.html">4885</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/StackWithTextChildren-mount.html">4915</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/SwatchColorPicker-mount_base.html">9417</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/SwatchColorPicker-mount.html">9466</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/TagPicker-mount_base.html">2360</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/TagPicker-mount.html">2341</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/TeachingBubble-mount_base.html">79946</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/TeachingBubble-mount.html">80334</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Text-mount_base.html">808</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Text-mount.html">829</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/TextField-mount_base.html">1550</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/TextField-mount.html">1574</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/ThemeProvider-mount_base.html">1454</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/ThemeProvider-mount.html">1465</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/ThemeProvider-virtual-rerender_base.html">1129</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/ThemeProvider-virtual-rerender.html">1144</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2022</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1999</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Toggle-mount_base.html">1145</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/Toggle-mount.html">1136</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/buttonNative-mount_base.html">547</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27002/perf-test/buttonNative-mount.html">541</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by spmonahan** [comment]
@mbest I cannot add you as a reviewer but I'd appreciate your input on this.

Unfortunately, I've not been able to find a way to implement the early render that doesn't break the rendering of TilesList (which depends on List) so I'm making it opt-in here. I'm open to suggestions on how to make it the default behavior but I think this might be the safest course.

cc: @ThomasMichon

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

File: change/@fluentui-react-d174ae16-7e82-4494-b026-d256fe1eb378.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-react-d174ae16-7e82-4494-b026-d256fe1eb378.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 19 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27002&commitId=f561ed462b6becc0ee11d8d9a28c98d6a60224e2&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1028 screenshots unchanged, 0 screenshots with different dimensions and 17 screenshots with visible difference.


  <details><summary><b>unknown</b> 19 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ActivityItem.Compact.default.chromium.png|27| Changed|
|ActivityItem.Root - RTL.default.chromium.png|695| Changed|
|ActivityItem.Root.default.chromium.png|1083| Changed|
|Breadcrumb.Root - RTL.click.chromium.png|7334| Changed|
|Breadcrumb.Root - RTL.default.chromium.png|6862| Changed|
|Breadcrumb.Root - RTL.hover.chromium.png|7119| Changed|
|Breadcrumb.Root - RTL.itemLinkFocus.chromium.png|7211| Changed|
|Breadcrumb.Root - RTL.longTitleHover.chromium.png|7029| Changed|
|Breadcrumb.Root - RTL.overflowButtonFocus.chromium.png|6908| Changed|
|Breadcrumb.Root - RTL.shortTitleHover.chromium.png|6938| Changed|
|Breadcrumb.Root.click.chromium.png|7084| Changed|
|Breadcrumb.Root.default.chromium.png|7065| Changed|
|Breadcrumb.Root.hover.chromium.png|7337| Changed|
|Breadcrumb.Root.itemLinkFocus.chromium.png|7416| Changed|
|Breadcrumb.Root.longTitleHover.chromium.png|7253| Changed|
|Breadcrumb.Root.overflowButtonFocus.chromium.png|7118| Changed|
|Breadcrumb.Root.shortTitleHover.chromium.png|7199| Changed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

Comment (spmonahan) [reply change/@fluentui-react-d174ae16-7e82-4494-b026-d256fe1eb378.json:1]: Diff is primarily image assets not loading + Pivot which is known to trigger false positives.

**Pull request merged**